agent/mission_solver/solver.py

```
@@ -0,0 +1,137 @@
+
+    # 提取结果
+    plan = {}
+    total_ap = 0
+    total_runs = 0
+    details = {}
+
+    sol = h.getSolution()
+    col_values = sol.col_value
+
+    for i in range(fn):
+        count = int(ceil(col_values[i])) if i < len(col_values) else 0
```

Ruff (0.15.11)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)